### ISSUE TITLE
Crash when launching OMS script with Python

### ISSUE THREAD

**Issue by RoiArthurB** [openmole/openmole#451]
## Describe the bug

Hello,
I'm trying to use OpenMole in remote and it crash every time I try to run an experiment.

### My openmole arguments 

```bash
$ ./openmole  --mem 8G --port 8081 --remote --no-browser --unoptimizedJS --workspace ./workspace
```

I simply followed the Python tutorial and here's what I have :

### My python code 
```python
import sys

f = open("output.txt", 'w')
f.write(str(arg))
```

### My OMS code 
```oms
/// Declare the variable
val arg = Val[Int]
val output = Val[File]

// python task
val pythonTask =
  PythonTask(workDirectory / "hello.py") set (
    inputs += arg.mapped,
    outputs += arg,
    outputs += output mapped "output.txt",
  )

val copy = CopyFileHook(output, workDirectory / "result/hello${arg}.txt")
val env = LocalEnvironment(2)

DirectSampling(
  evaluation = pythonTask,
  sampling = arg in (0 to 10)
) hook copy on env
```

### My workspace content 
![image](https://user-images.githubusercontent.com/16764085/75645438-79f2af00-5c78-11ea-91a7-4f4d1d568eda.png)

### Error

When I run the OMS file, the web-browser crash and the server log display this : 
```logs
Mar 02, 2020 11:24:16 AM org.openmole.gui.server.core.Utils$ addPluginRoutes
INFO: Loading GUI plugins
Mar 02, 2020 11:24:16 AM org.openmole.ui.Application$ $anonfun$run$25
INFO: Server listening on port 8081.
java.lang.NoSuchMethodError: java.lang.String.lines()Ljava/util/stream/Stream;
	at org.openmole.gui.server.core.Execution.$anonfun$allStates$3(Execution.scala:276)
	at scala.concurrent.stm.ccstm.InTxnImpl.runBlock(InTxnImpl.scala:570)
	at scala.concurrent.stm.ccstm.InTxnImpl.nestedAttempt(InTxnImpl.scala:515)
	at scala.concurrent.stm.ccstm.InTxnImpl.trueNestedAtomicImpl(InTxnImpl.scala:371)
	at scala.concurrent.stm.ccstm.InTxnImpl.nestedAtomicImpl(InTxnImpl.scala:340)
	at scala.concurrent.stm.ccstm.InTxnImpl.atomic(InTxnImpl.scala:263)
	at scala.concurrent.stm.ccstm.CCSTMExecutor.apply(CCSTMExecutor.scala:24)
	at org.openmole.gui.server.core.Execution.outputStreamData$1(Execution.scala:275)
	at org.openmole.gui.server.core.Execution.$anonfun$allStates$4(Execution.scala:280)
	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:238)
	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
	at scala.collection.TraversableLike.map(TraversableLike.scala:238)
	at scala.collection.TraversableLike.map$(TraversableLike.scala:231)
	at scala.collection.AbstractTraversable.map(Traversable.scala:108)
	at org.openmole.gui.server.core.Execution.$anonfun$allStates$1(Execution.scala:279)
	at scala.concurrent.stm.ccstm.InTxnImpl.runBlock(InTxnImpl.scala:570)
	at scala.concurrent.stm.ccstm.InTxnImpl.topLevelAttempt(InTxnImpl.scala:526)
	at scala.concurrent.stm.ccstm.InTxnImpl.topLevelAtomicImpl(InTxnImpl.scala:398)
	at scala.concurrent.stm.ccstm.InTxnImpl.atomic(InTxnImpl.scala:261)
	at scala.concurrent.stm.ccstm.CCSTMExecutor.apply(CCSTMExecutor.scala:24)
	at org.openmole.gui.server.core.Execution.allStates(Execution.scala:272)
	at org.openmole.gui.server.core.ApiImpl.allStates(ApiImpl.scala:428)
	at org.openmole.gui.server.core.GUIServlet$$anonfun$apply$3.$anonfun$applyOrElse$159(GUIServlet.scala:138)
	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
	at scala.util.Success.$anonfun$map$1(Try.scala:255)
	at scala.util.Success.map(Try.scala:213)
	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
	at java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1402)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)

```

### Server
 - OS: Ubuntu 18.04.4 LTS
 - OpenMole version: 10.0 - Quadratic Quest (download archive from the website) 
### Client
 - OS: Manjaro 19.0.1
 - Browser version: Firefox 73.0.1 (with all plugin disabled)

**Comment by romainreuillon** [comment]
Hi Arthur,

this is because you JVM is too old. OpenMOLE requires a JVM version >= 9,

Romain

**Comment by RoiArthurB** [comment]
Hi, that was the problem. I install the latest OpenJDK version (openjdk 11) and it's working fine now.

**Comment by RoiArthurB** [comment]
Just to inform you, the official documentation ([here](https://openmole.org/Download.html#DownloadOpenMOLE)) says that Java 8 is fine for running OpenMole (which is not the case anymore with the v10).

![image](https://user-images.githubusercontent.com/16764085/75663230-103bca80-5ca3-11ea-8201-0b511507df02.png)

**Comment by romainreuillon** [comment]
Thanks, we'll fix that for version 10.1 and it is already fixed in the dev version: see next.openmole.org
